lib/core/src/lib/viewer/components/viewer.component.html

```diff
@@ -44,8 +44,8 @@
                         <mat-icon>navigate_before</mat-icon>
                     </button>
                     <img class="adf-viewer__mimeicon"
-                         [alt]="mimeType"
-                         [src]="mimeType | adfMimeTypeIcon"
+                         [alt]="(originalMimeType ? originalMimeType : mimeType)"
```

Comment:
can be just 
originalMimeType || mimeType

Reply:
As suggested, I have modified the changes